src/main/java/com/eternalcode/discordapp/review/GitHubReviewService.java

```diff
@@ -184,14 +184,37 @@ public void deleteMergedPullRequests(JDA jda) {
         }
     }
 
-    public void addUserToSystem(GitHubReviewUser gitHubReviewUser) {
+    public String addUserToSystem(GitHubReviewUser gitHubReviewUser) {
+        if (this.isUserExist(gitHubReviewUser)) {
+            return "User already exists";
+        }
+
         this.discordAppConfig.reviewSystem.reviewers.add(gitHubReviewUser);
         this.configManager.save(this.discordAppConfig);
+
+        return "User added";
     }
 
-    public void removeUserFromSystem(Long discordId) {
+    public String removeUserFromSystem(Long discordId) {
+        if (!this.isUserExist(discordId)) {
+            // User does not exist
+            return "User does not exist, nothing to remove";
+        }
+
         this.discordAppConfig.reviewSystem.reviewers.removeIf(user -> user.discordId().equals(discordId));
         this.configManager.save(this.discordAppConfig);
+
+        return "User removed";
+    }
+
+    private boolean isUserExist(Long discordId) {
+        return this.discordAppConfig.reviewSystem.reviewers.stream()
+            .anyMatch(user -> user.discordId().equals(discordId));
+    }
+
+    private boolean isUserExist(GitHubReviewUser gitHubReviewUser) {
+        return this.discordAppConfig.reviewSystem.reviewers.stream()
+            .anyMatch(user -> user.discordId().equals(gitHubReviewUser.discordId()));
```

Comment:
te metody powinny się raczej nazywać `userExists()`

Reply:
to jest boolean